python/sparkdl/transformers/param.py

```diff
@@ -91,12 +94,23 @@ def getOutputCol(self):
         """
         return self.getOrDefault(self.outputCol)
 
-
 # New in sparkdl
 
 class SparkDLTypeConverters(object):
 
     @staticmethod
+    def toTFGraph(value):
```

Comment:
Directly importing `tf.GraphDef` without the input tensor mapping is not working.
Thus the user either provides a `TensorFlow Graph` or provide a `GraphFunction` (for developers).